packages/client/src/pages/forum/components/AddTopic/index.tsx

```diff
@@ -33,9 +35,15 @@ export const AddTopic = () => {
     event.preventDefault();
     const data = new FormData(event.currentTarget);
 
+    const title = data.get('title')
+    const description = data.get('description')
+
+    if(title) {
```

Comment:
_Не критично, но лучше исправить_

```suggestion
    if (title) {
```